ssas/service/main/main.go

```diff
@@ -337,3 +350,38 @@ func closeRows(rows *sql.Rows) {
 func cliTrackingID() string {
 	return fmt.Sprintf("cli-command-%d", time.Now().Unix())
 }
+
+func getXData(clientID, APIkey string) {
```

Comment:
since the name is `getXData`, would it make sense for us to return the XData value and display it outside of the function?